package.json

```diff
@@ -42,7 +42,7 @@
     "chai": "^4.2.0",
     "chromedriver": "^89.0.0",
     "codecov": "^3.7.1",
-    "cypress": "^7.5.0",
+    "cypress": "8.0",
```

Comment:
This doesn't look right. NPM version numbers are usually three digits.

How did you install this? I'm hoping you ran `npm install cypress@8.0.0`

Reply:
Corrected the number versioning. Now it shows 8.0.0